routers/web/repo/setting/setting.go

```diff
@@ -488,6 +488,13 @@ func SettingsPost(ctx *context.Context) {
 			}
 		}
 
+		if form.ChangeDefaultWikiBranch != "" {
```

Comment:
It may be better for the user if the default wiki branch is always shown in the form but only changed if a different value is given. For example the username field is not empty when you open the settings and not empty.

Reply:
There is such a check.

![image](https://github.com/go-gitea/gitea/assets/2114189/dbe5bb28-c5c8-4f8f-abb6-6161eda36464)

Reply:
Don't really talking about a missing check but as user I think I would like to see the current branch name in the settings instead of an empty field.

Reply:
> Don't really talking about a missing check but as user I think I would like to see the current branch name in the settings instead of an empty field.

It is there. See the screenshot in the description.

![image](https://github.com/go-gitea/gitea/assets/2114189/b972c5ba-f51b-466a-9515-6851ba5b8da5)

Reply:
Oh, then sorry for the noise. I only saw the the `Change...` and `if field != ""`. Could you change the field name to `default_wiki_branch`? Every field in the form is implicit `change_...`.

Reply:
19d924af2baaf3781fd68d92f22cb5b6867988ee